config/brokers/mt-channel-broker/deployments/broker-filter.yaml

```diff
@@ -84,8 +84,19 @@ spec:
             value: knative.dev/internal/eventing
           - name: FILTER_PORT
             value: "8080"
+
+        resources:
+          requests:
+            cpu: 100m
+            memory: 100Mi
+          limits:
+            cpu: 1000m
+            memory: 1000Mi
```

Comment:
these are just made up, right? :smile:

Reply:
yeah just [the same ones we've had for a while in serving](https://github.com/knative/serving/blob/main/config/core/deployments/controller.yaml#L63) which seem to work ok